app/models/applicant.rb

```diff
@@ -115,11 +115,19 @@ def self.unflagged_applicants(admission)
   end
 
   class << self
+    def match_email?(field)
```

Comment:
Nå er ikke jeg noe Ruby-geni, men burde ikke disse metodene kun være definert ett sted?

Reply:
Du har helt rett. Jeg tenkte det ville gitt performance gains å slippe å måtte initialize Applicant modellen for å kjøre det. Men om noe så er det såpass minimalt uansett, hehe.

Reply:
Fixed